CHANGELOG.md

```diff
@@ -3,6 +3,12 @@ aws-parallelcluster-cookbook CHANGELOG
 
 This file is used to list changes made in each version of the AWS ParallelCluster cookbook.
 
+x.x.x
+------
+
+**BUG FIXES**
+- Fix the way `DirectoryService.DomainAddr` property is interpreted to set the SSSD property `ldap_uri`.
```

Comment:
What about: 
- Fix `DirectoryService.DomainAddr` conversion to `ldap_uri` SSSD property when it contains multiples domain addresses.

Reply:
I like it, updating in the next rev